lua/supermaven-nvim/commands.lua

```diff
@@ -5,10 +5,12 @@ local M = {}
 M.setup = function()
   vim.api.nvim_create_user_command("SupermavenStart", function()
     api.start()
+    vim.notify("Supermaven ON")
```

Comment:
Instead of using `vim.notify()`, require at the top `local log = require("supermaven.logger")` and use `log:info("the message")` so if the users have the option `log_level = "off"` it won't trigger.